zetaclient/types/event.go

```diff
@@ -1,7 +1,31 @@
 package types
 
 import (
+	"bytes"
+	"encoding/hex"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
 	"github.com/zeta-chain/node/pkg/coin"
+	"github.com/zeta-chain/node/pkg/constant"
+	"github.com/zeta-chain/node/pkg/crypto"
+	"github.com/zeta-chain/node/pkg/memo"
+	"github.com/zeta-chain/node/zetaclient/config"
+)
+
+// InboundCategory is an enum representing the category of an inbound event
+type InboundCategory int
+
+const (
+	// InboundCategoryGood represents a processable inbound
+	InboundCategoryGood InboundCategory = iota
```

Comment:
`0` should indicate an invalid category (default type value)

Reply:
https://github.com/zeta-chain/node/pull/3184/commits/97ce52a94a2553fccc9f93e5e7815c3745766281